1.8/administration/installing/cloud/aws/advanced/aws-custom.md

```diff
@@ -0,0 +1,68 @@
+---
+post_title: Generating Custom AWS CF Templates
+nav_title: Custom CF Template
+menu_order: 102
+---
+
+You can create custom advanced templates for DC/OS. You can then deploy and run DC/OS from your own private S3 bucket. 
+
+**Prerequisites:**
+
+* A node that meets custom installer bootstrap node [system requirements](/docs/1.8/administration/installing/custom/system-requirements/).
+* Advanced template [system requirements](/docs/1.8/administration/cloud/aws/advanced/system-requirements/).
+* An Amazon S3 bucket with read-write access.
+
+
+1.  Download the [dcos_generate_config.sh](https://dcos.io/releases/) to your bootstrap node.
+1.  Create a directory named `genconf` on your node and navigate to it.
+    
+    ```bash
+    $ mkdir -p genconf
+    ```
+1.  Create a configuration file and save as `config.yaml`. This configuration file specifies your AWS credentials and the S3 location to store the generated artifacts. These are the required parameters:
```

Comment:
Make sure to let the users know that config.yaml needs to be generated in genconf directory